### DISCUSSION TITLE
Prevent students from accessing SEB without SEB configuration file

### DISCUSSION THREAD

**Discussion by jonnypui** [SafeExamBrowser/seb-win-refactoring#117] [Q&A]
We run SEB with specific configuration files on Moodle LMS. Lately we've discovered students can directly access the exam without using our configuration file. Is there a method where we can force students to use our specific configuration file to access the examination?

**Comment by danschlet** [comment]
https://safeexambrowser.org/windows/win_usermanual_en.html#moodle
